packages/components/src/slider/styles.ts

```diff
@@ -0,0 +1,194 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/react';
+import type { CSSProperties } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { COLORS, CONFIG } from '../utils';
+import { flow } from '../utils/flow';
```

Comment:
Since we're exporting `flow` directly from the style mixins, could we rewrite this line to

```suggestion
import { flow } from '../utils';
```

?

Reply:
The flow util has been removed. See https://github.com/WordPress/gutenberg/pull/42315#discussion_r926315552